src_override/constants.F90

```diff
@@ -138,6 +138,7 @@ module constants_mod
 
 use shr_kind_mod,   only : R8 => shr_kind_r8
 use shr_const_mod,  only : &
+     SHR_CONST_PSTD,                      & ! standard pressure ~ pascals
```

Comment:
Should be PSTD_MKS => SHR_CONST_PSTD and below remove line 205.

Reply:
I thought of this but then I believe 206 would have to change to read

PSTD     = PSTD_MKS*10.0_r8 

I added one more variable to make the relationship of the 2 GFS variables to the single shared constant more clear.

Reply:
I'm happy to change it if you don't agree though.